features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -35,7 +35,8 @@ Feature: Gift Exchange Challenge
       And I am on "My Gift Exchange" gift exchange edit page
     When I check "Sign-up open?"
       And I submit
-    When I view open challenges
+    When all indexing jobs have been run
+      And I view open challenges
```

Comment:
If possible, it probably makes sense to give `I view open challenges` the same treatment we gave `I go to the collections page` -- we're pretty much always going to want to run indexing jobs before we go there so we're getting the latest info. If that works, then you should be able  to remove all of the `all indexing  jobs have been run` steps you had to add before `I view open challenges`.